docs/logging.asciidoc

```diff
@@ -0,0 +1,135 @@
+[[logging-integrations]]
+=== Logging integrations
+
+The Elastic APM agent provides integrations with both the default Python logging library,
+as well as http://www.structlog.org/en/stable/[`structlog`].
+
+[[logging]]
+==== `logging`
+
+For Python 3.2+, we use https://docs.python.org/3/library/logging.html#logging.setLogRecordFactory[`logging.setLogRecordFactory()`]
+to decorate the default LogRecordFactory to automatically add new attributes to
+each LogRecord object:
+
+* `elasticapm_transaction_id`
+* `elasticapm_trace_id`
+* `elasticapm_span_id`
+
+This factory also adds these fields to a dictionary attribute,
+`elasticapm_labels`, using the official ECS {ecs-ref}/ecs-tracing[tracing fields].
```

Comment:
@bmorelli25 this is failing the docs build, due to `current` vs `1.0` (the attribute resolves to 1.0, not current):

https://www.elastic.co/guide/en/ecs/current/ecs-tracing.html

https://www.elastic.co/guide/en/ecs/1.0/ecs-tracing.html

Reply:
Ahh good find. This is one of the problems we have with links to/from documentation books that are versioned differently. I've asked on Slack, but I don't see a way to link there with attributes right now. I say go back to the original link and I'll make a note to clean this up once we have a new `current` reference.

Reply:
Done.